app/styles/_site-chrome.scss

```diff
@@ -38,7 +37,175 @@
   .profile & {
     margin-bottom: 0;
   }
+}
+
+.sitechrome-topright {
+  display: flex;
+  align-items: flex-end;
```

Comment:
better to use the compass flexbox mixing for browser compat:

```sass
@include flexbox((
  display: flex,
  align-items: flex-end
));
```
(yes that's two parens)